### ISSUE TITLE
Traceback from get_orchestrator_build_logs()

### ISSUE THREAD

**Issue by twaugh** [projectatomic/osbs-client#702]
Should check that the return value from `get_build_logs()` is not None before attempting a string operation on it:

```
2017-10-11 10:43:28,546 platform:- - atomic_reactor.plugin - DEBUG - Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugin.py", line 242, in run
    plugin_response = plugin_instance.run()
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugins/exit_koji_import.py", line 447, in run
    koji_metadata, output_files = self.combine_metadata_fragments()
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugins/exit_koji_import.py", line 379, in combine_metadata_fragments
    for md in self.get_logs()]
  File "/usr/lib/python2.7/site-packages/atomic_reactor/plugins/exit_koji_import.py", line 181, in get_logs
    for entry in logs:
  File "/usr/lib/python2.7/site-packages/osbs/api.py", line 793, in get_orchestrator_build_logs
    for entry in logs.splitlines():
AttributeError: 'NoneType' object has no attribute 'splitlines'
```

(Seen internally in Koji task #14242107)

**Comment by lcarva** [comment]
Is `None` value an indication of another abnormality?

**Comment by twaugh** [comment]
It can mean the build is in a pending state:
https://github.com/projectatomic/osbs-client/blob/0.43/osbs/core.py#L466-L468

**Comment by yahzaa** [comment]
Related [PR](https://github.com/projectatomic/osbs-client/pull/721)

**Comment by lcarva** [comment]
Closed per #721
